_minutes/2021-08-19-wecg.md

```diff
@@ -0,0 +1,135 @@
+# WECG Meetings 2021, Public Notes—Aug 19, 2021
```

Comment:
```suggestion
# WECG Meetings 2021, Public Notes, Aug 19
```

for consistency with the other headings.

Reply:
done